lib/github/ldap.rb

```diff
@@ -180,10 +183,10 @@ def search(options, &block)
       instrument "search.github_ldap", options.dup do |payload|
         result =
           if options[:base]
-            @connection.search(options, &block)
+            forest_search(options, &block)
           else
             search_domains.each_with_object([]) do |base, result|
```

Comment:
Never seen this function to be called due to the loop logic to iterate over ```search_domains``` seems to be implemented outside, somewhere in GHE